### PR TITLE
Adding information mentioned on issue #5352

### PR DESCRIPTION
**Pull request by j0rt3g4**
Working on Issue #5352 
Copy review @nenonix @JohanFreelancer9 is required on this one guys. Thank you in advance!

### CODE REVIEW

File: windows/security/identity-protection/hello-for-business/hello-hybrid-cert-whfb-settings-adfs.md [hunk 1]
```diff
@@ -51,13 +51,16 @@ Sign-in the AD FS server with *Domain Admin* equivalent credentials.
 
 The Windows Hello for Business group provides the AD FS service with the permissions needed to enroll a Windows Hello for Business authentication certificate on behalf of the provisioning user.
 
+> [!TIP]
+> The account adfssvc is the ADFS Service account.
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/hello-for-business/hello-hybrid-cert-whfb-settings-adfs.md]:
```suggestion
> The adfssvc account is the AD FS service account.
```

File: windows/security/identity-protection/hello-for-business/hello-hybrid-cert-whfb-settings-adfs.md [hunk 2]
```diff
@@ -51,13 +51,16 @@ Sign-in the AD FS server with *Domain Admin* equivalent credentials.
...
+
 Sign-in a domain controller or management workstation with _Domain Admin_ equivalent credentials.
 
 1. Open **Active Directory Users and Computers**.
 2. Click the **Users** container in the navigation pane.
 3. Right-click **Windows Hello for Business Users** group
 4. Click the **Members** tab and click **Add**
-5. In the **Enter the object names to select** text box, type **adfssvc**.  Click **OK**.
+5. In the **Enter the object names to select** text box, type **adfssvc** or substitute the name of the ADFS Service account in your ADFS deployment.  Click **OK**.
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/hello-for-business/hello-hybrid-cert-whfb-settings-adfs.md]:
```suggestion
5. In the **Enter the object names to select** text box, type **adfssvc** or substitute the name of the AD FS service account in your AD FS deployment.  Click **OK**.
```

**Comment by j0rt3g4** [comment]
@JohanFreelancer9  Thank you! I've added the corrections

**Comment by mypil** [comment]
@mapalko - PR has been copy edited and is ready for final review. This is based on issue #5352

Thank you.

**Comment by illfated** [comment]
Very well. We need a final follow-up from the document manager, Daniel Simpson (or someone else on merge management duty).

**Comment by mypil** [comment]
@Dansimp - Can you possibly merge this PR as approved by @mapalko? 

Thank you.

@kenwith

**Comment by illfated** [comment]
#sign-off

**Comment by mypil** [comment]
@Dansimp - We would like to follow up. Can you please merge this approved PR?

Thank you.

**Pull request merged**